src/helix-ui/elements/HXTooltipElement.js

```diff
@@ -0,0 +1,162 @@
+import { HXElement } from './HXElement';
+import { getPositionWithArrow } from '../util';
+import debounce from 'lodash/debounce';
+import shadowStyles from './_hx-tooltip.less';
+
+const tagName = 'hx-tooltip';
+const template = document.createElement('template');
+
+template.innerHTML = `
+  <style>${shadowStyles}</style>
+  <div id="container" class="position-arrow">
+    <slot></slot>
+  </div>`;
+
+export class HXTooltipElement extends HXElement {
+    static get is () {
+        return tagName;
+    }
+
+    static get observedAttributes () {
+        return [ 'open' ];
+    }
+
+    constructor () {
+        super(tagName, template);
+        this._show = this._show.bind(this);
+        this._hide = this._hide.bind(this);
+        this._toggle = this._toggle.bind(this);
+        this._setPosition = this._setPosition.bind(this);
+        this._closeOnBackgroundClick = this._closeOnBackgroundClick.bind(this);
+    }
+
+    connectedCallback () {
+        if (!this.position) {
```

Comment:
Given that `position` reflects to the attribute, you can replace this entire if with the `$defaultAttribute()` function provided by HXElement.

```javascript
this.$defaultAttribute('position', 'top');
```

Reply:
👍